versions.gradle

```diff
@@ -6,6 +6,5 @@ ext {
     minSdkVersion = 15
     targetSdkVersion = 26
     compileSdkVersion = 26
-    buildToolsVersion = '26.0.1'
```

Comment:
Just curious, what's the change for this one? We don't need to specify build tools version anymore?

Reply:
Yes, it even triggers a linter if version mismatches with what android uses now.